File: Java Spring Boot Project/CompanyDirectory/src/main/java/com/mindthecode/CompanyDirectory/models/responses/AllEmployeesResponse.java
package com.mindthecode.CompanyDirectory.models.responses;

import java.util.List;

public class AllEmployeesResponse {

    private List<EmployeeResponse> employees;

    public AllEmployeesResponse(List<EmployeeResponse> employees) {
        this.employees = employees;
    }

    public List<EmployeeResponse> getEmployees() {
        return employees;
    }

    public void setEmployees(List<EmployeeResponse> employees) {
        this.employees = employees;
    }
}

File: Java Spring Boot Project/company_directory_app/lib/models/responses/all_employees_response.dart
import 'package:company_directory_app/models/responses/employee_response.dart';

class AllEmployeesResponse {
  final List<EmployeeResponse> employees;

  AllEmployeesResponse({this.employees});

  factory AllEmployeesResponse.fromJson(Map<String, dynamic> json) {
    var list = json['employees'] as List;
    List<EmployeeResponse> itemsList = list.map((i) => EmployeeResponse.fromJson(i)).toList();

    return AllEmployeesResponse(employees: itemsList);
  }
}
